designs/jane/src/body.mjs

```diff
@@ -0,0 +1,219 @@
+    paths,
+    Snippet,
+    snippets,
+    sa,
+    paperless,
+    macro,
+    measurements,
+    part,
+  }) {
+    //body
+    const lengthBody =
+      (measurements.waistToKnee + measurements.hpsToWaistBack) * (1 + options.shiftLength)
+
+    const workingHip =
+      measurements.chest > measurements.hips ? measurements.chest / 2 : measurements.hips / 2
+
+    const widthBody = workingHip * (1 + options.bodyEase)
+
+    const maxLength =
+      lengthBody > measurements.waistToFloor + measurements.hpsToWaistBack
+        ? measurements.waistToFloor + measurements.hpsToWaistBack
+        : lengthBody
+
+    points.topLeft = new Point(0, 0)
+    points.topRight = new Point(widthBody, 0)
+    points.bottomLeft = new Point(0, maxLength )
+    points.bottomRight = new Point(widthBody, maxLength )
+
+    points.gorestartLeft = points.bottomLeft.shiftTowards(points.topLeft, maxLength / 2)
+    points.gorestartRight = points.bottomRight.shiftTowards(points.topRight, maxLength / 2)
+
+    points.middle = points.topLeft.shiftTowards(points.topRight, widthBody / 2)
+    points.leftShoulder = points.middle.shiftTowards(
+      points.topLeft,
+      measurements.shoulderToShoulder / 2
+    )
+    points.rightShoulder = points.middle.shiftTowards(
+      points.topRight,
+      measurements.shoulderToShoulder / 2
+    )
+
+    paths.sideseam = new Path()
+      .move(points.topRight)
+      .line(points.topLeft)
+      .line(points.bottomLeft)
+      .line(points.bottomRight)
+      .line(points.topRight)
+      .close()
+
+    paths.leftGore = new Path()
+      .move(points.leftShoulder)
+      .line(points.gorestartLeft)
+      .addClass('fabric dashed')
+      .close()
+
+    paths.rightGore = new Path()
+      .move(points.rightShoulder)
+      .line(points.gorestartRight)
+      .addClass('fabric dashed')
+      .close()
+
+    //neckline
+
+    const neckWidth = measurements.neck * options.neckWidth
+
+    points.neckLeft = points.middle.shiftTowards(points.leftShoulder, neckWidth / 2)
+    points.neckRight = points.middle.shiftTowards(points.rightShoulder, neckWidth / 2)
+
+    points.middleHem = points.bottomLeft.shiftTowards(points.bottomRight, widthBody / 2)
+
+    points.neckFront = points.middle
+      .shiftTowards(points.middleHem, measurements.hpsToBust * options.neckDepthFront)
+      .attr('data-text', 'Front Neckline')
+      .attr('data-text-class', 'center')
+
+    points.neckCp1 = points.neckRight.shift(90, points.neckFront.dy(points.neckRight) * 0.8)
+    points.neckCp2 = points.neckFront.shift(180, points.neckFront.dy(points.neckRight) * 0.8)
+
+    points.neckCp3 = points.neckLeft.shift(90, points.neckFront.dy(points.neckLeft) * 0.8)
+    points.neckCp4 = points.neckFront.shift(0, points.neckFront.dy(points.neckLeft) * 0.8)
+
+    paths.neckLineFront = new Path()
+    .move(points.neckRight)
+      .curve(points.neckCp1, points.neckCp2, points.neckFront)
+      .curve(points.neckCp4, points.neckCp3, points.neckLeft)
```

Comment:
All paths should be styled. Without any classes, we have no idea of knowing what this is.